app/scripts/lib/buy-url.js

```diff
@@ -57,16 +57,16 @@ const createWyrePurchaseUrl = async (walletAddress, chainId) => {
  *
  * @param {string} walletAddress - Ethereum destination address
  * @param {string} chainId - Current chain ID
+ * @param {string|undefined} symbol - Token symbol to buy
  * @returns String
  */
-const createTransakUrl = (walletAddress, chainId) => {
+const createTransakUrl = (walletAddress, chainId, symbol) => {
   const { transakCurrencies, network } = BUYABLE_CHAINS_MAP[chainId];
 
   const queryParams = new URLSearchParams({
     apiKey: TRANSAK_API_KEY,
     hostURL: 'https://metamask.io',
-    cryptoCurrencyList: transakCurrencies.join(','),
-    defaultCryptoCurrency: transakCurrencies[0],
+    defaultCryptoCurrency: symbol || transakCurrencies[0],
```

Comment:
With this change, the transakCurrencies array is now pointless, we should use nativeCurrency I think?

Reply:
hi brad, took your suggestion and changed this:
`defaultCryptoCurrency: symbol || transakCurrencies[0],`
for this:
`defaultCryptoCurrency: symbol || nativeCurrency,`

but the array is not pointless actually, is used as a list of supported currencies that vary depending on the chainId
so having the list, allows us to check if XYZ token can be bought through transak (in this case)

went with the change still so we dont have to worry to keep the nativeCurrency on the first place in the array